eng/common/scripts/Verify-Resource-Ref.ps1

```diff
@@ -0,0 +1,40 @@
+
+. (Join-Path $PSScriptRoot common.ps1)
+Install-Module -Name powershell-yaml -RequiredVersion 0.4.1 -Force -Scope CurrentUser
```

Comment:
I know we chatted about this but we should probably remove this module import as well and move to checking it in.

Reply:
I plan to do that as part of the [publish tools work ](https://github.com/Azure/azure-sdk-tools/issues/119#event-3604992537) then I'll switch all instances of this over.